uid2_client/publisher_client.py

```diff
@@ -1,7 +1,4 @@
-"""Internal module for holding the Uid2Client class.
-
-Do not use this module directly, import through uid2_client module instead, e.g.
```

Comment:
Why was this comment removed?

Reply:
The UID2Client class is being deprecated and participants can use this directly. The [publisher guide](https://unifiedid.com/docs/sdks/uid2-sdk-ref-python#usage-for-publishers) mentions this